packages/theme/stories/introduction.stories.mdx

```diff
@@ -218,12 +218,12 @@ We have already mentioned that components will reference Theme variables, the ex
 - All UI Toolkit components have a CSS class on the root element, of the form `uitk<ComponentName>`. This is a rule with no exceptions.
 - The value from a CSS API variable (more on this below) is assigned as a style by using `var(--<variableName>-<...>);`.
 - A CSS variable reference can define a fallback value to be used if the primary variable does not have a value:
-  var(--primary-variable-value, var(--fallback-value))
+  `var(--primary-variable-value, var(--fallback-value))`
 - The fallback value can equally be a simple constant CSS value
-  var(--primary-variable-value, 12px);
+  `var(--primary-variable-value, 12px);`
 
 In the Button example above, font-size is being set to whatever value has been assigned to the variable `--uitkButton-fontSize`. If this variable has not been defined, or is not in scope at the point where this Button element is rendered, then the fallback value will be taken from `--uitk-text-fontSize`. Providing that a UI Toolkit theme is in scope at this point, this will have a value. If neither of these variables had values associated with them at this location, then the font-size would default to whatever value was in effect by the normal CSS mechanisms.
 
-Note the naming convention for the first variable `--uitkButton-fontSize`. The prefix matches exactly (even in case) the CSS class of the component. That identifies this as a `CSS API variable`. These are not actually assigned any value anywhere in the UI Toolkit code. They are a customisation mechanism intended for use by consumers of Toolkit components. A default (i.e. fallback) value is _always_ provided in this case, and will often reference a theme characteristic variable. If consumers need to override a default style assignment, these API variables provide a simple means to do this. The group of API variables available with each component are published as part of the docs for each component (as are the CSS class names).
+Note the naming convention for the first variable `--uitkButton-fontSize`. The prefix matches exactly (even in case) the CSS class of the component. That identifies this as a *CSS API variable*. These are not actually assigned any value anywhere in the UI Toolkit code. They are a customisation mechanism intended for use by consumers of Toolkit components. A default (i.e. fallback) value is _always_ provided in this case, and will often reference a theme characteristic variable. If consumers need to override a default style assignment, these API variables provide a simple means to do this. The group of API variables available with each component are published as part of the docs for each component (as are the CSS class names).
```

Comment:
```suggestion
Note the naming convention for the first variable `--uitkButton-fontSize`. The prefix matches exactly (even in case) the CSS class of the component. That identifies this as a *CSS API variable*. These are not actually assigned any value anywhere in the UI Toolkit code. They are a customization mechanism intended for use by consumers of Toolkit components. A default (i.e. fallback) value is _always_ provided in this case, and will often reference a theme characteristic variable. If consumers need to override a default style assignment, these API variables provide a simple means to do this. The group of API variables available with each component are published as part of the docs for each component (as are the CSS class names).
```